addons/website_sale/views/templates.xml

```diff
@@ -1320,37 +1473,38 @@
                     method="get"
                     t-att-action="keep(attribute_value=0, tags=0)"
                 >
+                    <t
+                        t-set="visible_attributes"
+                        t-value="len([a for a in attributes if a.value_ids and len(a.value_ids) &gt; 1])"
+                    />
```

Comment:
This is not applied in offcanvas and I think the offcanvas could benefit from the `t-cache` also. Isn't it possible to factorise the form and its content (or at the very least only the content)?